.evergreen/install-composer.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+set -o errexit  # Exit the script with error if any of the commands fail
```

Comment:
```suggestion
set -o errexit  # Exit the script with error if any of the commands fail

# Supported environment variables
DEPENDENCIES=${DEPENDENCIES:-} # Specify "lowest" to prefer lowest dependencies
COMPOSER_FLAGS="${COMPOSER_FLAGS:-}" # Optional, additional Composer flags
```

I suppose you could omit `COMPOSER_FLAGS` since it's not used anywhere and really just set below when we process `$DEPENDENCIES`.

I know `$PHP_PATH` and `$PROJECT_DIRECTORY` are also referenced below, but `$PROJECT_DIRECTORY` is always just assumed. I'm not sure where `$PHP_PATH` comes from as that was originally exported by `install-dependencies.sh` but now handled by the `locate PHP binaries` function.

Reply:
`$PHP_PATH` is set through the `add_expansions_to_env` setting in the `install composer` function. `locate PHP binaries` is responsible for finding the path to PHP binaries and creates the evergreen expansion.